gadget/gadget.go

```diff
@@ -454,7 +542,7 @@ func validateRole(vs *VolumeStructure, vol *Volume) error {
 			return fmt.Errorf(`role of this kind must have an implicit label or "writable", not %q`, vs.Label)
 		}
 	case "mbr":
-		if vs.Size > 446 {
+		if vs.Size > SizeMBR {
```

Comment:
Can the MBR be smaller? Is there a smallest MBR?

Reply:
See my previous comment. The structure can be smaller than 446 bytes, though it's probably not useful.